packages/server-core/src/user/avatar/avatar.hooks.ts

```diff
@@ -155,6 +155,28 @@ const updateUserAvatars = async (context: HookContext<AvatarService>) => {
   }
 }
 
+const addAndSortByUserName = async (context: HookContext<AvatarService>) => {
+  if (!context.params.query) return
+
+  const userNameAlias = 'userName'
+
+  const sort = context.params.query.$sort?.['owner']
+  delete context.params.query.$sort?.['owner']
+  const nameLike = context.params.query.name as any
```

Comment:
Here we should be using `context.params.query.name.$like`? Also can we avoid `as any`?